share/skel/cpanfile

```diff
@@ -1,9 +1,37 @@
 requires "Dancer2" => "[d2% dancer_version %2d]";
 
-recommends "YAML"             => "0";
-recommends "URL::Encode::XS"  => "0";
-recommends "CGI::Deurl::XS"   => "0";
-recommends "HTTP::Parser::XS" => "0";
+recommends "YAML"                    => "0";
+recommends "URL::Encode::XS"         => "0";
+recommends "CGI::Deurl::XS"          => "0";
+recommends "HTTP::Parser::XS"        => "0";
```

Comment:
I don't see HTTP::Parser as a dep of either D2 or Plack, so I don't think this should be here.

Reply:
Done!